src/main/kotlin/app/nush/bot/commands/Projects.kt

```diff
@@ -99,9 +103,57 @@ object Projects : Command {
                         )
                     )
                     reply("Channel <#${channel.id}> created")
+
+                    if (!(words.size == 4 && words[3].equals("channel-only"))) {
+                        val discordWebhook =
+                            bot.clientStore.channels[channel.id].createWebhook(CreateWebhook("For GitHub"))
+                        createWH(projName, discordWebhook.id, discordWebhook.token)
+                        reply("GitHub repository $projName created")
+                        reply("Link: https://github.com/appventure-nush/$projName")
+                    }
+                }
+                command("linkrepo") {
+                    val guildId = guildId ?: return@command
+                    val roles =
+                        clientStore.guilds[guildId].getMember(authorId).roleIds
+                    if (config.excoRole !in roles && !config.dev) {
+                        reply("You are not authorized")
+                        return@command
+                    }
+                    if (words.size < 3) {
+                        reply("Please specify a project name")
+                        return@command
+                    }
+                    val projName = words[2]
+                    val discordWebhook = bot.clientStore.channels[channelId].createWebhook(CreateWebhook("For GitHub"))
+                    try {
+                        createWH(projName, discordWebhook.id, discordWebhook.token)
+                        reply("Successfully linked #${channel.get().name} with $projName")
+                    } catch (e: IOException) {
+                        reply("Repository not found")
+                        return@command
+                    }
+
                 }
             }
         }
     }
 
+    fun createWH(projName: String, id: String, token: String) {
```

Comment:
Should separate webhook creation and project creation. Consider scenario when user runs link command but types repo name wrongly.